boards/arm/arduino_giga/doc/index.rst

```diff
@@ -0,0 +1,156 @@
+.. _arduino_giga_board:
```

Comment:
Image is too large, please use something like https://tinypng.com/

Reply:
Donez.